### PR TITLE
Feature: Add OpenTracing

### PR DESCRIPTION
**Pull request by bketelsen**
This change adds global OpenTracing support.  See the home.go HomeHandler for an example of how to use it.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - sleep 15
+  build:
+    image: golang:1.10
+    commands:
+      - go test ./...
+
+services:
+  mongo:
+    image: mongo:3.0
+    command: [ --smallfiles ]
```
Comment (arschles) [inline .drone.yml]: can you add a badge or link to where the drone page is? even better would be to add PR statuses from drone if possible

File: Gopkg.lock [hunk 2]
```diff
@@ -498,6 +571,37 @@
   revision = "12b6f73e6084dad08a7c6e575284b177ecafbc71"
   version = "v1.2.1"
 
+[[projects]]
+<<<<<<< HEAD
```
Comment (arschles) [inline Gopkg.lock]: can you fix these merge conflicts?

File: docker-compose.yml [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+  mysql:
+    image: bitnami/mysql:5.7.21-r7
+    ports:
+      - "3306:3306"
+    environment:
+      - "ALLOW_EMPTY_PASSWORD=yes"
+      - "MYSQL_USER=vgp"
+      - "MYSQL_PASSWORD=vgp"
+      - "MYSQL_DATABASE=vgoprox"
+  jaeger:
```
Comment (arschles) [inline docker-compose.yml]: 🎉

File: .drone.yml (hunk 1 above)
Comment (bketelsen) [reply .drone.yml]: Yes but it has to be enabled before I can add them so separate PR.

File: Gopkg.lock (hunk 2 above)
Comment (bketelsen) [reply Gopkg.lock]: fixed - d'oh!

**Comment by bketelsen** [comment]
PTAL

**Comment by bketelsen** [comment]
Fixes #88

**Comment by arschles** [comment]
@bketelsen FYI if you put the `Fixes #123` into the OP it'll auto-close the issue on merge

**Pull request merged**